web3swift.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
     spec.name         = 'web3swift'
-    spec.version      = '2.5.1'
+    spec.version      = '2.6.0'
```

Comment:
I have no idea why on earth GitHub is flagging this as a change being made here, as `develop` is `2.6.0` not `2.5.1`. This popped up as a result of updating the PR to the latest version of `develop` when `2.6.0` was released.